src/main.rs

```diff
@@ -32,6 +37,23 @@ impl Handler {
     }
 }
 
+async fn start_ranking_watcher(ctx: Context) {
+    let ranking_watcher_channel_id = ChannelId(
+        env::var("RANKING_WATCHER_CHANNEL_ID")
```

Comment:
Given the increasing number of such env variables, I was thinking about moving to some toml/yaml configuration file.

It's definitely outside of the scope for this PR though, just an idea for later.